src/jsonrpc/server/server.rs

```diff
@@ -123,9 +123,16 @@ impl StratisParams {
                     false,
                 )))
             }
-            StratisParamType::PoolUnlock(unlock_method, uuid) => {
-                Ok(StratisRet::PoolUnlock(stratis_result_to_return(
-                    pool::pool_unlock(engine, unlock_method, uuid, self.fd_opt).await,
+            StratisParamType::PoolStart(uuid, unlock_method) => {
+                Ok(StratisRet::PoolStart(stratis_result_to_return(
```

Comment:
Might be a rebase error...should have ```expects_fd...false``` ?

Reply:
This is intentional. We don't need that macro when the file descriptor is optional.